composer.json

```diff
@@ -32,40 +32,17 @@
         }
 
     ],
-    "require-dev": {
-        "doctrine/coding-standard": "^12.0.0",
-        "phpstan/phpstan": "^1.10.60",
-        "phpstan/phpstan-phpunit": "^1.3.16",
-        "phpunit/phpunit": "^10.5.12",
-        "vimeo/psalm": "5.23.0",
-        "roave/infection-static-analysis-plugin": "^1.34.0"
-    },
     "autoload": {
         "psr-4": {
             "Roave\\BetterReflection\\": "src"
         }
     },
-    "autoload-dev": {
```

Comment:
This will also mess with editors 🤔

I'm not convinced of moving everything, but I'm open to trying it, for now 🤔